### PR TITLE
Update pylintrc to the latest version

### PR DESCRIPTION
**Pull request by mstajdohar**

### CODE REVIEW

**Comment by mstajdohar** [comment]
I have also fixed some issues that made pylint fail on Python 2. We only test pylint with Python 3 on Travis.

File: .pylintrc [hunk 1]
```diff
@@ -156,65 +157,42 @@ class-rgx=[A-Z_][a-zA-Z0-9]+$
...
-# expressions are accepted.
-generated-members=REQUEST,acl_users,aq_parent,objects,id,_instance,_class_name,FULL_PERMISSIONS,isoformat,get_full_name,is_authenticated,throttle_check,method_check,_meta,wrap_view,fields,cleaned_data,build_bundle,_headers,DoesNotExist
+# Maximum number of nested blocks for function / method body
+max-nested-blocks=5
 
 
 [FORMAT]
 
 # Maximum number of characters on a single line.
-# Django coding style guidelines allow up to 119 characters
```
Comment (tjanez) [inline .pylintrc]: We want to keep this comment, please re-add it.

**Comment by tjanez** [comment]
Please, augment the commit message (the body, not the summary) and tell that you fixed additional Pylint issues. I'm wondering why the issues are only seen on Python 2...

Generally speaking, the `.pylintrc` configuration file is still too complex, I hope they improve that in upcoming Pylint 2.0.

**Comment by tjanez** [comment]
Otherwise, LGTM

**Pull request merged**